.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -81,19 +81,19 @@ body:
     attributes:
       label: Anything in the Supervisor logs that might be useful for us?
       description: >
-        Supervisor Logs can be found in [Settings -> System -> Logs](https://my.home-assistant.io/redirect/logs/)
-        then choose `Supervisor` in the top right.
+        Supervisor Logs can be found in [Settings -> System -> Logs](https://my.home-assistant.io/redirect/logs/?provider=supervisor)
+        then choose `Supervisor` in the top right. Alternatively enter `supervisor logs` in the Home Assistant CLI.
```

When running from SSH/web terminal add-on, you have to use `ha` in front. I'd prefer if we always prepend the commands with `ha`.

```suggestion
        then choose `Supervisor` in the top right. Alternatively enter `ha supervisor logs` in the Home Assistant CLI.
```

---

I tried to address it by explicitly saying "from Home Assistant CLI". But you are right it may still be confusing to beginners, and using `ha` in HA CLI just raises a warning, so it doesn't do much harm.